invokeai/app/invocations/latent.py

```diff
@@ -143,9 +143,9 @@ class TextToLatentsInvocation(BaseInvocation):
     type: Literal["t2l"] = "t2l"
 
     # Inputs
-    # TODO: consider making prompt optional to enable providing prompt through a link
     # fmt: off
-    prompt: Optional[str] = Field(description="The prompt to generate an image from")
+    conditioning: Optional[ConditioningField] = Field(description="Conditioning for generation")
+    seed:        int = Field(default=-1,ge=-1, le=np.iinfo(np.uint32).max, description="The seed to use (-1 for a random seed)", )
```

Comment:
We've removed seed from this node (since it's only used in the noise node).

Reply:
Ah... A bit wrong merge conflict resolve. I'l delete it.